### ISSUE TITLE
Scrolling support

### ISSUE THREAD

**Issue by tsani** [kevin-li-195/zeno#3]
I figure something like mod+shift+{h,j,k,l} can be used to issue a scroll in that direction.

**Comment by kevin-li-195** [comment]
Oh that's not a bad idea; but what about page down and up? Or are you thinking that mod+shift+{h,j,k,l} could issue a smaller scroll (i.e. one scroll wheel click)?

On 01/11/2017 02:12 PM, Jacob Thomas Errington wrote:

I figure something like mod+shift+{h,j,k,l} can be used to issue a scroll in that direction.

**Comment by tsani** [comment]
Yeah, I'm thinking a smaller scroll amount. Page up/down is too much.
